app/modules/favorites/favorites.controller.js

```diff
@@ -0,0 +1,22 @@
+'use strict';
+
+require('./favorites.module.js')
+    .controller('FavoritesController', FavoritesController);
+
+// @ngInject
+function FavoritesController(FavoritesFactory) {
+    var vm = this;
+    vm.favorites = getFavSubjects();
+
+    function getFavSubjects() {
+        console.log('CONTROLLER: getFavSubjects')
+        return FavoritesFactory.list()
+            .then(function (collection) {
+                var favSubjects = collection[0].links.subjects;
+                console.log('FavoritesController: ', favSubjects);
+                return favSubjects;
```

Comment:
this should probably be all in the factory - the controller shouldn't care how it gets stuff from the API